Dockerfile

```diff
@@ -35,7 +35,9 @@ RUN go get -d -v ./...
 
 ARG APP_VERSION
 
-RUN CGO_ENABLED=1 GOOS=linux go install -tags="blst_enabled,jemalloc,allocator" -ldflags "-X main.Version=`if [ ! -z "${APP_VERSION}" ]; then echo $APP_VERSION; else git describe --tags $(git rev-list --tags --max-count=1); fi` -linkmode external -extldflags \"-static -lm\"" ./cmd/ssvnode
+RUN git fetch --tags
+RUN if [ ! -z "${APP_VERSION}" ]; then echo $APP_VERSION; else echo `git describe --always --tags $(git rev-list --tags --max-count=1)`; fi
```

Comment:
Why do we have 2 RUN commands

Reply:
@amir-blox The first RUN fetches information about tags, which is needed for `git describe`. The second RUN debug-prints the version, so we can check it in the pipeline, it will make debugging easier if passing version gets broken

Reply:
removed the second one